### DISCUSSION TITLE
UI issue: main area height

### DISCUSSION THREAD

**Discussion by polkan-msk** [dbeaver/dbeaver#22026] [Q&A]
Hi everyone,
Does anybody know how to fix 'non full height' issue of the App's main area? 
![2023-11-29_10-03](https://github.com/dbeaver/dbeaver/assets/7592144/07971ea1-484b-4c08-94a1-ea66eb469733)
I got this after I rearranged tables to have them both opened one above another.
I do not know when exactly that empty area appeared (I rearranged tables several times), but now I can't get rid of it. I can resize the main area, but it is not possible to drag its top edge higher than it is displayed on my screen shot. 
Please advise.

**Comment by LonwoLonwo** [comment]
Hello @polkan-msk 

Maybe [Reset Settings](https://github.com/dbeaver/dbeaver/wiki/Reset-UI-settings) or [Reset workspace](https://github.com/dbeaver/dbeaver/wiki/Reset-workspace) will be helpful in your case.

**Reply by polkan-msk** [reply]
hi @LonwoLonwo 
It works. Thank you!
